src/main/client/src/components/app/side-bar/map-layers/map-layers-all/MapLayersAllView.js

```diff
@@ -130,33 +139,18 @@ const MapLayersAllView = ({
                             >
                                 {lg.layers.filter(layer => (nestedVal(layer.relations.find(r => r), ['relationType'], '') !== 'link'))
                                     .sort((a, b) => b.layerOrder - a.layerOrder)
-                                    .map(l => (
-                                        layerList.find(layer => layer.id === l.id
-                                            && !layer.parentLayer
-                                            && !layerList.some(ll => ll.parentLayer === l.id))
-                                            ? (
-                                                <MapLayerContainer
-                                                    inputDisabled={l._source === 'shapefile'}
-                                                    key={l.id}
-                                                    layer={l}
-                                                    layerGroupName={lg.name}
-                                                    handleLayerClick={handleLayerClick}
-                                                    checked={nestedVal(layerList.find(layer => layer.id === l.id), ['active'], false)}
-                                                />
-                                            )
-                                            : (
-                                                <SubLayerContainer
-                                                    key={l.id}
-                                                    layer={l}
-                                                    handleLayerClick={handleLayerClick}
-                                                    activeSubGroups={activeSubGroups}
-                                                    handleSubGroupClick={handleSubGroupClick}
-                                                    /* eslint-disable-next-line max-len */
-                                                    handleSubLayerGroupClick={handleSubLayerGroupClick}
-                                                    layersToFind={layersToFind}
-                                                    checkboxSquare={checkboxSquare}
-                                                />
-                                            )
+                                    .map(l => (layerList.find(layer => layer.id === l.id
```

Comment:
Uncaught TypeError: Cannot read property 'find' of undefined
    in MapLayersAllView (at MapLayersAll.js:142)
    in MapLayersActive (created by ConnectFunction)
    in ConnectFunction (at MapLayersView.js:135)

Lisää Shapetiedosto ja mene Kaikki -välilehteen
Sama ongelma myös kun lisää uuden Tason

Reply:
Tämän ei pitäisi tapahtua enää, kun tehtävässä on muutettu react-beautiful-dnd kirjaston versiota.

Reply:
Tämä bugi ei liity tähän tehtävään, on mukana devissä.

Tehty kuitenkin nopea fixi, jotta sovellus ei kaadu.